### ISSUE TITLE
Alt text for Images

### ISSUE THREAD

**Issue by tkacheva** [magento/magento2-page-builder#746]
h1. Story

As a marketer I want to set alt text to images (image, banner, slide) *so that* content I create with PageBuilder has better accessibility and SEO 

h2. Business Value
make PageBuilder content accessible and SEO optimized

AC:
- SEO(alt and title) block should be available for Slider and Banner PB entity
- instead of `ALT` attribute should be used [`aria-label`](https://www.aditus.io/aria/aria-label/)

Testing notes:
- need to be sure that `aria-label` works as `alt` attribute. Can be read by a screen reader
Possible tools:
- https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn
- https://chrome.google.com/webstore/detail/images-onoff/nfmlhilnjccdggifdbhnhkffmjgalbgg

**Comment by manavluhar** [comment]
@magento I am working on this

**Comment by sidolov** [comment]
@magento export issue to Jira project PB as Story
